src/Recipe.js

```diff
@@ -44,10 +46,10 @@ class Recipe {
     let sum = 0;
 
     this.ingredients.forEach(ingredient => {
-      sum += ingredient.amount * ingredient.costPerUnit;
+      sum += ingredient.totalCostInCents;
```

Comment:
I see that you took out the multiplication by the quantity (`ingredient.amount`). I know we need that multiplication somewhere, and I see now that that multiplication is being done in the `formatIngredient.totalCostInCents` line, so it seems like a good call to delete it here so we're not doing it twice. Makes me wonder if the initial tests were flawed though. Let's discuss when we meet.

Reply:
it was already done in the method that sets up the recipe format